Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -3598,4 +3602,39 @@ - (void)updateProgress:(CGFloat)progress OfWaitTask:(ORKWaitStepViewController *
     }
 }
 
+#pragma mark - Location Task
+
+- (IBAction)locationButtonTapped:(id)sender {
+    [self beginTaskWithIdentifier:LocationTaskIdentifier];
+}
+
+- (ORKOrderedTask *)makeLocationTask {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    
+    ORKInstructionStep *step1 = [[ORKInstructionStep alloc] initWithIdentifier:@"locationTask.step1"];
+    step1.title = @"Location Survey";
+    [steps addObject:step1];
+    
+    // Location question with current location observing on
+    ORKQuestionStep *step2 = [[ORKQuestionStep alloc] initWithIdentifier:@"locationTask.step2"];
+    step2.title = @"Where are you right now?";
+    step2.answerFormat = [[ORKLocationAnswerFormat alloc] init];
+    [steps addObject:step2];
+    
+    // Location question with current location observing off
+    ORKQuestionStep *step3 = [[ORKQuestionStep alloc] initWithIdentifier:@"locationTask.step3"];
+    step3.title = @"Where is your home?";
+    ORKLocationAnswerFormat* locationAnswerFormat  = [[ORKLocationAnswerFormat alloc] init];
+    locationAnswerFormat.useCurrentLocation= NO;
+    step3.answerFormat = locationAnswerFormat;
+    [steps addObject:step3];
+    
+    ORKCompletionStep *step4 = [[ORKCompletionStep alloc] initWithIdentifier:@"locationTask.step4"];
+    step4.title = @"Survey Complete";
+    [steps addObject:step4];
+    
+    ORKOrderedTask *locationTask = [[ORKOrderedTask alloc] initWithIdentifier:LocationTaskIdentifier steps:steps];
+    return locationTask;
+}
```

Comment:
Did you want to leave the location question at the end of Selection Survey still?